src/site/content/en/blog/serial/index.md

```diff
@@ -323,18 +323,19 @@ async function readUntilClosed() {
     }
   }
 
-  await port.close();
+  return port.close();
```

Comment:
This still returns a promise. Did you remove `await` because you don't care about the promise? Since someone might infer behavior from the code instead of reading the text, I'd suggest adding a comment that you don't care about the returned promise.

Reply:
I think it's clearer this way.
See https://jakearchibald.com/2017/await-vs-return-vs-return-await/

Reply:
It's clear if you know that `close()` returns a promise and that your script doesn't need it. If I'm reading this article, it's presumed that I don't know anything about the api or the method.

Reply:
Since `close()` returns `Promise<undefined>` I prefer using `await` over `return` to highlight the fact that it is an asynchronous operation.

Reply:
Addressed.